src/visitpy/common/visitmodule.C

```diff
@@ -1306,23 +1306,27 @@ CreateDictionaryFromDBOptions(DBOptionsAttributes &opts)
             PyDict_SetItemString(dict,name,PyString_FromString(opts.GetString(name).c_str()));
             break;
           case DBOptionsAttributes::Enum:
-            // If you modify this section, also check the Enum case in
-            // FillDBOptionsFromDictionary
-            int enumIndex = opts.GetEnum(name);
-            stringVector enumStrings = opts.GetEnumStrings(name);
-            std::string itemString(enumStrings[enumIndex]);
-            if (enumStrings.size() > 1)
             {
```

Comment:
The compiler was warning about missing cases in switch statement. I added `default` case but that initially failed because the local vars in this block were not scoped to their specific case.